practices/java-web/HelloWorldFilterWebApplication/src/main/java/com/dowa/java/web/filters/StatisticFilter.java

```diff
@@ -0,0 +1,40 @@
+package com.dowa.java.web.filters;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+
+public class StatisticFilter implements Filter {
+    private int indexVisitNum = 0;
```

Comment:
this is too specific for each url, what if I add a new URL? how would you change this code such that no matter if I add a new url this filter still works